proto/v1/discovery.proto

```diff
@@ -52,24 +52,39 @@ message DiscoveryOccurrence {
   // Analysis status for a resource. Currently for initial analysis only (not
   // updated in continuous analysis).
   enum AnalysisStatus {
+    option allow_alias = true;
     // Unknown.
     ANALYSIS_STATUS_UNSPECIFIED = 0;
     // Resource is known but no action has been taken yet.
     PENDING = 1;
     // Resource is being analyzed.
     SCANNING = 2;
-    // Analysis has finished successfully.
+    // Analysis has finished successfully. 
     FINISHED_SUCCESS = 3;
+    // Analysis has completed
+    COMPLETE = 3;
     // Analysis has finished unsuccessfully, the analysis itself is in a bad
     // state.
     FINISHED_FAILED = 4;
-    // The resource is known not to be supported
+    // The resource is known not to be supported.
     FINISHED_UNSUPPORTED = 5;
   }
 
   // The status of discovery for the resource.
   AnalysisStatus analysis_status = 2;
 
+  // Indicates which analysis completed successfully. Multiple types of
+  // analysis can be performed on a single resource.
+  message AnalysisCompleted {
+    repeated string analysis_type = 1;
+  }
+
+  AnalysisCompleted analysis_completed = 7;
+
+  // Indicates any errors encountered during analysis of a resource. There
+  // could be 0 or more of these errors.
+  repeated google.rpc.Status analysis_error = 8;
```

Comment:
Should this be a message with the analysis type included as well so we know which analysis failed?

Reply:
the Status proto (https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto) has a details field which will allow us to populate information regarding which analysis failed into it.